plugin/trino-iceberg/pom.xml

```diff
@@ -301,34 +301,70 @@
         </dependency>
     </dependencies>
 
-    <build>
-        <pluginManagement>
-            <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-enforcer-plugin</artifactId>
-                    <configuration>
-                        <rules>
-                            <requireUpperBoundDeps>
-                                <excludes combine.children="append">
-                                    <exclude>com.google.guava:guava</exclude>
```

Comment:
Why this appears as a change here?
did you mean a prep commit?

Reply:
It is verbatim move from <build> -><pluginManagement> to default profile

Reply:
I extracted separate commit. But it still shows in diff as it is different level of indentation.